geojson_pydantic/features.py

```diff
@@ -44,23 +46,22 @@ def __geo_interface__(self) -> Dict[str, Any]:
             "geometry": self.geometry.__geo_interface__
             if self.geometry is not None
             else None,
+            "properties": self.properties,
```

Comment:
properties should always be set to at least `{}`

Reply:
Technically GeoJSON can have null properties. But [__geo_interface__](https://gist.github.com/sgillies/2217756#__geo_interface__) says

> properties (optional) - A mapping of feature properties ...

So, I would say leave this how it was.

Reply:
the issue then is that you wouldn't be able to do `Feature(**Feature(type="Feature", geometry=None, properties=None).__geo_interface__)`

Reply:
🤔 https://github.com/jazzband/geojson/blob/main/geojson/feature.py#L31 always default to `{}`

Reply:
That is an interesting one. Not being able to put it back into itself is problematic. And `None` becomes `{}` in `__geo_interface__` that would mean `properties` would change when fed back in. The least bad thing might be a pre validator that converts `None` to `dict()`? 

> Note: we already allow geometry to be None in the geo_interface while the spec says it should be a mapping 🤷

The current way won't add the `geometry` key to the dict unless it is truthy. So it would never be `None`, it just wouldn't exist at all.

Reply:
> The current way won't add the geometry key to the dict unless it is truthy. So it would never be None, it just wouldn't exist at all.

```python
Feature(type="Feature", geometry=None, properties=None).__geo_interface__
>> {'type': 'Feature', 'geometry': None, 'properties': None}
```
we do add `geometry: None` in the current implementation

Reply:
@vincentsarago Sorry, I had started to edit that comment and apparently never saved it. Got lost in the 12 tabs open looking at too many parts at the same time. 

Based on a strict reading geometry may be wrong. However [jazzband/geojson](https://github.com/jazzband/geojson/blob/c8a489a0add8ea624a48e3fefb48f5604b77afcb/geojson/base.py#L64) is just returning self as the `__geo_interface__` so geometry could be `None` there as well. 

Which actually seems like a good idea, its supposed to be modeled after geojson, so just return `self.dict()`. Don't worry about picking fields like it currently does.